docs/user_guide.md

```diff
@@ -1008,10 +1117,47 @@ If you set a quota on a project, this quota cannot be exceeded. The quota is app
 
 **WARNING**: You cannot revert a rule after you run it. It is strongly recommended to perform a dry run before you run rules. 
 
-To modify an existing rule, click the three vertical dots next to a rule to disable, edit, or delete that rule. 
+To modify an existing rule, use the **Action** drop-down menu next to a rule to disable, edit, or delete that rule. 
 
 ![Modify tag retention rules](img/tag-retention5.png)
 
+## Configure Tag Immutability
+
+By default, users can repeatedly push an image with the same tag to repositories in Harbor. This causes the previous image to effectively be overwritten with each push, in that the tag now points to a different image and the image that previously used the tag now becomes tagless. This is due to the Docker implementation, that does not enforce the mapping between an image tag and the image digest. This can be undesirable in certain cases, because the tag can no longer be trusted to identify the image version. The sha256 digest remains reliable and always points to the same build, but it is not rendered in a human-friendly format.  
+
+Moreover, the Docker implementation requires that deleting a tag results in the deletion of all other tags that point to the same digest, causing unwanted image deletions.
+
+To prevent this, Harbor allows you to configure tag immutability at the project level, so that images with certain tags cannot be pushed into Harbor if their tags match existing tags. This prevents existing images from being overwritten. Tag immutability guarantees that an immutable tagged image cannot be deleted, and cannot be altered through repushing, retagging, or replication.
+
+1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+1. Go to **Projects**, select a project, click the **More** ellipsis (`...`), and select **Tag Strategy**.
```

Comment:
The UI seems not v1.10, it should be captured from `latest`  as the `...` is introduced after v1.10.

Reply:
Argh, looks like I jumped the gun on this. Thanks for spotting it @wy65701436 ! I will revert the doc changes and screen caps.

Reply:
Fixed